UKHomeOffice/cerberus-service
# COP-8018: Delete test generated tasks after tests completed

## Description
Delete all the tasks generated by automation tests. 
Get all the tasks of `assignTarget `
Loop over all the tasks and get the processes with `movementID` attribute.
Check the value of the  movementID and if it contains `AUTOTEST` then delete the process.

## To Test
npm run cypress:test:report -- -b electron
Once tests completed,  check there are no tasks with name starts with "AUTOTEST-".

## Developer Checklist

\* Required
